junit5-extension/src/main/java/io/kroxylicious/testing/kafka/junit5ext/Topic.java

```diff
@@ -13,7 +13,7 @@
 
 /**
  * Signals the wish for a topic to be created and injected into a test as either test parameter
- * or field.
+ * or field. The name will conform to the pattern '[a-z]+_[a-z]+'.
```

Comment:
wondering if this is useful, maybe it should just say that tests should assume that any characters that are allowed by Kafka could be used in the name and have a link off to the annotation saying "if you need to control the naming, use this annotation".

Reply:
I'd mention the annotation as you suggest.